msprime/__init__.py

```diff
@@ -71,6 +71,9 @@
     JC69,
     MatrixMutationModel,
     MutationModel,
+    SMM_MicrosatMutationModel,
```

Comment:
so `SMM` is probably well enough known on it's own. Many of the others that this generalized system I implemented might not be. E.g. EL2. I doubt it will be _that_ confusing to users however as there will be no name clash with the AA or nucleotide models